app/controllers/events_controller.rb

```diff
@@ -4,7 +4,7 @@ class EventsController < ApplicationController
   before_action :set_event, only: [:show, :edit, :update, :destroy]
 
   def index
-    @events = Event.published.newest_first
```

Comment:
I was thinking we might want to show the current event (and upcoming events) on the index page. The navbar should get swapped to say "All Events" if we do that.

Reply:
Gotcha, that's cool, @Kentasmic and I can / will swap that out now